docs/quick_start/installing.rst

```diff
@@ -198,3 +198,64 @@ Also you need to change the value of Storage plugin. See |Configure Storage Plug
     }
 
 Now, it's time to restart Fledge. Thereafter you will see Fledge is running with PostgreSQL.
+
+
+Using Docker Containerizer to install Fledge
+#############################################
+
+Fledge is provided a private registry. No auth/TLS yet on this private registry for docker container.
+
+Useful instructions are given below for it's setup:
+
+- Edit the daemon.json file, whose default location is /etc/docker/daemon.json on Linux, If the daemon.json file does not exist, create it. Assuming there are no other settings in the file, it should have the following contents:
+
+.. code-block:: console
+
+    { "insecure-registries":["52.3.255.136:5000"] }
+
+- Restart Docker for the changes to take effect. sudo systemctl restart docker.service
+
+- Check using "sudo docker info" command, you should have following in output:
```

Comment:
Check using the command

.. code-block:: console

     sudo docker info

You should see the following output: